style.css

```diff
@@ -545,6 +688,57 @@ body {
     font-size: 2.3rem;
   }
 
+  .modal-info {
+    width: 80%;
+    line-height: 25px;
+    padding: 3rem;
+    height: 100%;
+    margin-top: 0;
+    margin-bottom: 12rem;
+  }
+
+  .modal-dialog {
+    width: 150rem;
+    height: 500rem;
+  }
+
+  .modal-top-content {
+    margin-bottom: 20rem;
+  }
```

Comment:
Great job @GordonOtieno 🤝

- [ ] [Optional] As you can see, your desktop popup seems to be missing the close button. There is no way the user will navigate back to the main page if this button is not present. You may kindly fix this change since it is essential for this project
![button](https://user-images.githubusercontent.com/63917725/180194320-af02f340-686f-46e6-988d-a535c40b279e.PNG)